javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md

```diff
@@ -24,9 +24,7 @@ links:
 ---
 ## Content
 
-Number benefits from a number of new methods saving you from writing your own potentially error prone implementation.
-
-There are a large number so here are some of the ones that are likely to have more use:
+`Number` benefits from many new methods saving you from writing your own potentially error prone-implementation. Because of the increased number of methods available, this insight will explain the ones you are more likely to use.
```

Comment:
```suggestion
The `Number` class benefits from many new methods saving you from writing your own potentially error prone-implementation. Because of the increased number of methods available, this insight will explain the ones you are more likely to use.
```